source/diagrams/manage-diagrams/clone-diagrams.txt

```diff
@@ -0,0 +1,13 @@
+.. _clone_diagrams:
```

Comment:
I have been prefacing my refs with `rm_` because it is possible one day the refs will collide when we add additional inventory files to https://github.com/mongodb/docs-relational-migrator/blob/master/snooty.toml#L4-L7. Would you mind doing this through out the additional pages?

```suggestion
.. _rm_clone_diagrams:
```

Reply:
Done!